packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -503,22 +506,28 @@ Future<void> buildGradleApp({
   );
 
   // Call size analyzer if --analyze-size flag was provided.
-  if (buildInfo.analyzeSize != null && !globals.platform.isWindows) {
+  if (buildInfo.codeSizeDirectory != null && !globals.platform.isWindows) {
     final SizeAnalyzer sizeAnalyzer = SizeAnalyzer(
       fileSystem: globals.fs,
       logger: globals.logger,
       processUtils: ProcessUtils.instance,
     );
+    final String archName = getNameForAndroidArch(androidBuildInfo.targetArchs.single);
+    final File aotSnapshot = globals.fs.directory(buildInfo.codeSizeDirectory)
```

Comment:
oh I thought you called it codeSizeDirectory just for the AOT. If you're gonna have this flag for everything, perhaps outputSizeDirectory? Since there's more than just code that you're breaking down.

Reply:
You didn't reply to this one :)